synapse/http/server.py

```diff
@@ -569,6 +569,9 @@ def render_GET(self, request: Request) -> bytes:
         set_clickjacking_protection_headers(request)
         return super().render_GET(request)
 
+    def directoryListing(self) -> resource.NoResource:
+        return self.childNotFound
```

Comment:
I wonder if we should use `twisted.web.pages.notFound` if available due to https://github.com/advisories/GHSA-vg46-2rrj-3647?

(`childNotFound` == `_UnsafeNoResource`)

Reply:
I did an update to `notFound`.